### ISSUE TITLE
Create accessibility/external link policy landscape analysis

### ISSUE THREAD

**Issue by amycole501** [GSA/usagov-benefits-eligibility#908]
perform current landscape analysis for federal accessibility statement and link policy examples especially focusing on external link handling and "warning". We are not responsible for third-party site accessibility and BEARS visitors need to know we cannot be held responsible for applications on third-party sites for benefits.

CDC https://www.cdc.gov/Other/disclaimer.html 
EAC https://www.eac.gov/voter_resources/resources_for_voters_with_disabilities.aspx

**Comment by EnriqueOrt** [comment]
@amycole501 to sync to Ernie to cover material.

**Comment by EnriqueOrt** [comment]
As of 08/07 no external links yet. TBD, does not need review until 2024
